src/ATen/native/xpu/sycl/MultinomialKernel.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include <ATen/ATen.h>
+
+namespace at::native::xpu {
+
+void multinomial_with_replacement_kernel(
```

Comment:
`multinomial_kernel`. Since you have two kernels in the function. `with_replacement` and `sample_once`.

Reply:
done